typescript/styling-engine.ts

```diff
@@ -0,0 +1,51 @@
+import { HeadingElement } from './forms'
+
+export type FormStyle = BackgroundStyle & {
+  /** The font family to use for the entire form page */
+  fontFamily?: string
+  /** The container that houses the form elements */
+  formContainer?: BorderStyle &
+    BackgroundStyle & {
+      /** In pixels */
+      maxWidth?: number
```

Comment:
to follow on your discussion with Blake about supporting other units
```suggestion
      maxWidth?: {unit: 'pixels', value: number}
```

Reply:
Yep, definitely could. Just not sure whether anyone will care, or if they will just adjust until it looks right in any unit.